src/CSV.jl

```diff
@@ -38,7 +38,7 @@ Read and parses a delimited file, materializing directly using the `sink` functi
 """
 function read(source, sink=nothing; copycols::Bool=false, kwargs...)
     if sink === nothing
-        @warn "`CSV.read(input; kw...)` is deprecated in favor of `CSV.read(input, DataFrame; kw...)"
+        @warn "`CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)"
```

Comment:
Yes, but the plan is to remove DataFrames from within CSV, so I've corrected the warning here that you should explicitly call `using DataFrames` yourself to load it.